build.sh

```diff
@@ -17,24 +17,33 @@ rpmostree_deps="rpm-ostree selinux-policy-targeted rpm-build"
 curl -L --remote-name-all https://kojipkgs.fedoraproject.org//packages/rpm-ostree/2018.7/1.fc28/x86_64/rpm-ostree-{,libs-}2018.7-1.fc28.x86_64.rpm
 dnf -y install ./rpm-ostree*.rpm && rm -f *.rpm
 
-# These are only used to build things in here
+# These are only used to build things in here, we define them separately because
+# they're cleaned up later
 self_builddeps="cargo golang"
-build_tools="make ${self_builddeps} git rpm-build"
-virtinstall_deps="libvirt libguestfs-tools qemu-kvm /usr/bin/qemu-img /usr/bin/virsh /usr/bin/virt-install"
-releng_scripts_deps="rsync pygobject3-base python3-gobject-base"
-# To support recursive containerization and manipulating images
-container_tools="podman buildah skopeo"
-misc_tools="jq awscli"
```

Comment:
They're right here, we were just missing `${misc_tools}` in the list below right?

Reply:
Right, that was the motivation for changing it to using `xargs`. That way we can directly comment in the same place as we pass them to `dnf`.